File: uri/c/2684.c
#include <stdio.h>
#include <string.h>

//incompleto
int main()
{

    int n, k;
    char tags[201], aux[50];

    scanf("%d\n", &n);

    for (int i = 0; i < n; i++)
    {

        scanf("%d", &k);
        fgets(tags, sizeof(tags), stdin);
        tags[strlen(tags) - 1] = '\0';

        if (k == 0)
        {
            for (size_t s = 0; s < strlen(tags); s++)
            {
                int aberto, fechado;
                aberto = fechado = 1;

                if ((strcmp(tags[s], "<") == 0) & (aberto == 0))
                {
                    printf("error\n");
                    /*setar s para tamanho da string para sair da repetição*/
                    s = strlen(tags);
                }
                else
                {
                    aberto == 0;
                }

                if ((strcmp(tags[s], ">") == 0) & (fechado == 0))
                {
                    printf("error\n");
                    s = strlen(tags);
                }
                else
                {
                    fechado == 0;
                }

                if ((aberto == fechado == 0) & ((s + 1) == strlen(tags)))
                {
                    printf("Successful!!");
                }

                if ((aberto == fechado == 0))
                {
                    aberto == fechado == 1;
                }
            }
        }
        else
        {
            /* code */
        }
    }

    return 0;
}

File: uri/python/2684.py
n = int(input())
for i in range(n):

    temp = input()
    k = int(temp[0])
    tags = temp[2:len(temp)]
    tags.strip(" ")

    # debusg
    print(tags)

    if(tags[0] != "<" or tags[-1] != ">"):
        print("error")
    else:
        if(k == 1):
            # if("<" in tags[1:len(tags)-1] or ">" in tags[1:len(tags)-1]):
            #   print("error")
            # else:
            aux = tags.count("<", 1, len(tags)-1) + \
                tags.count(">", 1, len(tags)-1)
            print(aux)
            if(aux//2 == 0):
                aberto = True
                for j in range(1, len(tags)-1):
                    if (tags[j] == "<" and aberto == True):
                        print("error")
                        j = len(tags)
                    elif(tags[j] == ">" and aberto == False):
                        print("error")
                    elif (tags[j] == ">"):
                        aberto = False
